src/ui/components/shared/SharingModal/SharingModal.tsx

```diff
@@ -143,7 +143,9 @@ function SharingModal({ recording, hideModal }: SharingModalProps) {
             <div className="flex flex-col space-y-1 overflow-hidden">
               <div className="font-bold">Privacy Settings</div>
               <PrivacyDropdown {...{ recording }} />
-              <SecurityWarnings operations={recording.operations} onClick={() => {}} />
+              {recording.operations ? (
+                <SecurityWarnings operations={recording.operations} onClick={() => {}} />
```

Comment:
The fix is fine but I'm concerned that TypeScript didn't catch this. Do you know why?

Reply:
Good catch — `operations` was not typed correctly. Fixed the type so that it's optional, and handled the null case where TS threw errors.

Reply:
🙇🏼 Thanks!